content/src/content/jcr_root/apps/core/wcm/components/contentfragmentlist/v1/contentfragmentlist/_cq_dialog/.content.xml

```diff
@@ -53,6 +53,14 @@
                                         fieldLabel="Tags"
                                         multiple="{Boolean}true"
                                         name="./tagNames"/>
+                                    <maxItems
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                        fieldDescription="Maximum number of items displayed in list. Empty for all items."
+                                        fieldLabel="Max Items"
+                                        min="{Long}0"
+                                        value="{Long}10"
```

Comment:
I would not set the default limit to 10, the default should be empty (all items).

Reply:
If we do this I would be +1 to merge.

Reply:
I went ahead and added this change directly to the `development` branch:
https://github.com/adobe/aem-core-wcm-components/commit/003391ea93f9044302d98b8fb2de64b47b002a31